src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  padding: 0;
 }
+
+.logo img {
+  max-height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  font-weight: 500;
+  min-width: 1024px;
+  position: relative;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0, 5);
+  font-family: Roboto, sans-serif;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: block;
+  text-align: center;
+  font-size: 12px;
+  margin-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-right: 0;
+}
+
+.nav__list:first-child {
+  margin-left: 0;
+}
+
+.nav__list:last-child {
+  margin-right: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+[data-qa='hover']:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+/* .nav__link:hover {
+  text-decoration: underline;
+  text-decoration-color: #38a1f6;
+  text-decoration-thickness: 2px;
+} */
```

Comment:
The hover effect for '.nav__link' is commented out. If this effect is intended to be part of the design, consider uncommenting and adjusting it to fit the overall style.